src/main/java/uk/gov/mca/beacons/service/model/Beacon.java

```diff
@@ -25,6 +27,7 @@
   private UUID id;
 
   @Enumerated(EnumType.STRING)
+  @NotNull
```

Comment:
Ensures that beaconType is always set. 

This must be true for all beacons registered